onnxmltools/convert/keras/operator_converters/Dropout.py

```diff
@@ -0,0 +1,19 @@
+# -------------------------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License. See License.txt in the project root for
+# license information.
+# --------------------------------------------------------------------------
+
+from keras.layers import Dropout, SpatialDropout1D, SpatialDropout2D, SpatialDropout3D
+
+from ...common._registration import register_converter
+from ...common._apply_operation import apply_identity

```

Comment:
Please swap`*_reg*` and `*_apply*`. Thanks.

Reply:
switched